docs/appendix/contributing.md

```diff
@@ -52,6 +52,126 @@ When in doubt, ask during review.
+  It’s useful to have a real or imagined why question to serve as a prompt.
+
+  {{% expand "Click to view template" %}}
+
+  ```md
+  # Concept
+
+  Introductory paragraph. Possibly containing further subsections or links to relevant conceptual information.
+
+  ## Subconcept
+
+  Paragraphs containing explanation. Add imagery as needed.
+
+  (possibly more Subconcept sections)
+
+  ### Do x with Subconcept (optional)
+
+  Information for what to do with the concept (possibly linking to How-tos or containing how-tos).
+
+  ## Next Steps
+
+  Links to related content.
+  ```
+
+  {{% /expand%}}
+
+- **How-to Guides (procedural)**: A task-oriented piece of content that directs a reader to perform actions step by step to complete a task, like a recipe. Generally starts with a description of the task and things to consider and then provides a set of numbered steps to follow. For example, the [Installation page](https://docs.viam.com/get-started/installation/) or the [Find module page](https://docs.viam.com/registry/configure/).
+
+  {{% expand "Click to view template" %}}
+
+  ```md
+  # Do this task
+
+  Description of task. Possibly containing further subsections.
+
+  ## Do x
+
+  1. Ordered list of actions to perform. Written in imperative form. Add imagery as needed.
+
+  (possibly more Do X sections)
+
+  ## Next Steps
+
+  Links to related content.
+  ```
+
+  {{% /expand%}}
+
+- **Tutorials**: A learning-oriented piece of content that functions as a lesson for the reader. A tutorial helps readers to learn skills by doing something meaningful and attainable.
+
+  {{% expand "Click to view template" %}}
+
+  ```md
+  # Do X with Y
+
+  Outline the why.
+  Tell the story of the machine.
+
+  ## Requirements
+
+  What does the reader need to already know.
+  What will you be using (hardware/software).
+
+  ## Build X
+
+  Build steps
+
+  ## Configure your X
+
+  Configuration steps
+
+  ## Program your X
+
+  Code and directions.
+
+  ## Next steps
+
+  Link to other tutorials with cards or text.
+
+  {{< cards >}}
+  {{% card link="/tutorials/get-started/blink-an-led" %}}
+  {{< /cards >}}
+  ```
+
+  For the full template see [template.md](https://github.com/viamrobotics/docs/blob/main/docs/tutorials/template.md).
+
+  {{% /expand%}}
+
+- **Reference**: A concise, information-oriented piece of content that generally starts with an overview/introduction and then a list of some kind (configuration options, api methods, etc.).
+  Examples include the [API pages](https://docs.viam.com/build/program/apis/) as well as [component and service pages](https://docs.viam.com/build/configure/components/arm/).
+
+  Example template: [Component template](https://github.com/viamrobotics/docs/blob/main/docs/build/configure/components/component/_index.md).
+
+  {{% expand "Click to view template" %}}
+
+  ```md
+  # Product, Feature or API Name
+
+  Description or introduction. Possibly containing further subsections.
+
+  ## List or table of items
```

Comment:
What would this title actually be? It wouldn't be "List of API Methods"... would it be i.e. "API Methods" or should this heading actually not be here at all?

Reply:
Examples:
- https://docs.viam.com/build/configure/components/arm/#api
- https://docs.viam.com/build/program/apis/data-client/#api

On the configure page there is no heading: https://docs.viam.com/build/configure/

```suggestion
  ## List or table of items (heading optional as needed)
```